cmd/node/node.go

```diff
@@ -644,6 +654,8 @@ func (app *SpacemeshApp) initServices(nodeID types.NodeID,
 	app.oracle = blockOracle
 	app.txProcessor = processor
 	app.atxDb = atxdb
+	app.fetch = fetcher
```

Comment:
if `fetch` isn't `Start'ed or `Close`ed maybe we don't have to keep it in the `app` struct?